run_remove_js.sh

```diff
@@ -0,0 +1,16 @@
+# ./node_modules/.bin/tsc
+# node debugstart.js
+# remove the create .js file
```

Comment:
Why is this needed? If it is, we should add it as `make clean`

Reply:
Yes, I think we might need to have make clean. We meet some problem when trying to fix the testing code issue.